lib/http_server_test_fixture/mock_handlers/mock_not_found.ex

```diff
@@ -0,0 +1,8 @@
+defmodule HTTPServerTestFixture.Handlers.MockNotFound do
+  @behaviour HTTPServer.Handler
+
+  @impl HTTPServer.Handler
+  def handle(req) do
+    {404, "The requested URL #{req.path} was not found on this server. Sorry!"}
+  end
```

Comment:
Like with the HEAD story, is there a way to accomplish this without having to add a `handle()` implementation to every single Handler?  I would think there would just be a default for 404 in general, that would perhaps just call out to `lib/handlers/not_found.ex` or even just have the 404 response generated directly in the default case?

Reply:
Just pushed up my edit that removes the redundant Page Not Found handlers to use the `HTTPServer.Handlers.NotFound` handler instead!